examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.java

```diff
@@ -97,7 +101,8 @@ public ChipDeviceController currentCommissioner() {
   /////////// Command Interface /////////
   @Override
   public void run() throws Exception {
-    maybeSetUpStack();
+    // TODO: setup chip storage from Java, currently it is using example one from chip-tool
+    // maybeSetUpStack();
     runCommand();
     maybeTearDownStack();
```

Comment:
We also need to comment out maybeTearDownStack() since we has commented out maybeSetUpStack

Reply:
agree. done, thanks